apps/src/applab/applab.js

```diff
@@ -123,6 +119,10 @@ var twitterOptions = {
 var vizAppWidth = 400;
 var VIZ_APP_HEIGHT = 400;
 
+function stepDelayFromStepSpeed(stepSpeed) {
```

Comment:
Not so random, since this is the only file still using it.  I might move it down about 300 lines though, so it's close to the place it gets used.